src/languageServer.ts

```diff
@@ -32,11 +32,12 @@ export function activate(context: ExtensionContext) {
 	};
 
 	let clientOptions: LanguageClientOptions = {
-		documentSelector: ['ddl'],
+		documentSelector: ['ddl', 'sql'],
 		synchronize: {
 			configurationSection: ['ddl'],
```

Comment:
Does this also have to change?

Reply:
No. If I do that then it will override the documentselector for the sql extension and that extension then stops working.

Reply:
So what's the difference between the configurationSection and the documentSelector in this case? Are we just sharing the configuration between two file types now?

Reply:
documentSelector registers this LSP for a given document type / language. The sql language is defined outside of our extension and just reused if available. 
For configurationSection I am a bit puzzled. It seems to be some kind of vscode config reference but I did not find useful information yet. The only thing I can tell you is that once I put in 'sql' then the sql extension does no longer work.

Reply:
Looks like maybe it's about the settings based on this comment - https://github.com/microsoft/vscode-languageserver-node-example/blob/1fd1ee0d52151564ac16a8ffdbe736492223624d/client/src/extension.ts#L32

Reply:
There's a little more here - https://vscode-docs.readthedocs.io/en/stable/extensions/example-language-server/

Reply:
Doesn't sound like I want to do that for sql.

Reply:
btw just as a side note:

![image](https://user-images.githubusercontent.com/435853/81184330-9381e100-8fb0-11ea-81c5-fe62b5308788.png)

Reply:
Seems we may want to spin this configurationSection bit off into its own issue and deal with it separately? WDYT @apupier ?

Reply:
agree